internal/resolve/catalog.go

```diff
@@ -117,7 +117,7 @@ func (r *CatalogResolver) Resolve(ctx context.Context, ext *ocv1.ClusterExtensio
 			predicates = append(predicates, successorPredicate)
 		}
 
-		// Apply the predicates to get the candidate bundles
+		// Store the predicates to get the candidate bundles
```

Comment:
IDE refactor gone wrong:
```suggestion
		// Apply the predicates to get the candidate bundles
```